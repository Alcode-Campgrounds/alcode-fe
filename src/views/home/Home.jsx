import React from 'react'

export default function Home() {
    return (
       <> 
            <img  className='hero-img' src='../hero.jpg' alt="serene campsite by a lake" />
        </>
        )
    }