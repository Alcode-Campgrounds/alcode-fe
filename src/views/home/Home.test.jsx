import React from 'react';
import { render } from '@testing-library/react';

import Home from './Home';

it('should render the Home component', () => {
  const { container } = render( <Home /> );

  expect(container).toMatchSnapshot();
});
