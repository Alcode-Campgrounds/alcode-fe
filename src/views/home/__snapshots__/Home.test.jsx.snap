// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Home component 1`] = `
<div>
  <h2>
    Landing Page
  </h2>
</div>
`;
